### ISSUE TITLE
`devtools::document()` not working with the latest commit

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/roxygen2#909]
I am working with the development version of `roxygen2` and with the latest commit (40cb4a6e8b149b4bbcc13bd7c674c222c26161aa), I can't seem to get `roxygenize` to work.

Here are the details about the error-

```r
> devtools::document()
Updating ggstatsplot documentation
Error in secs[[1]] : subscript out of bounds

> traceback()
18: mdxml_children_to_rd_top(mdxml, state)
17: markdown(text, tag)
16: markdown_if_active(pieces[, 2], x)
15: trim_docstring(markdown_if_active(pieces[, 2], x))
14: registry[[x$tag]](x)
13: FUN(X[[i]], ...)
12: lapply(tokens, parse_tag, registry = registry)
11: compact(lapply(tokens, parse_tag, registry = registry))
10: parse_tags(tokens, registry = registry, global_options = global_options)
9: .f(call = .l[[1L]][[1L]], srcref = .l[[2L]][[1L]], tokens = .l[[3L]][[1L]], 
       ...)
8: purrr::pmap(list(call = as.list(parsed)[has_tokens], srcref = refs[has_tokens], 
       tokens = tokens[has_tokens]), block_create, registry = registry, 
       global_options = global_options)
7: FUN(X[[i]], ...)
6: lapply(files, tokenize_file, registry = registry, global_options = global_options)
5: parse_package(base_path, env = NULL, registry = registry, global_options = options)
4: roxygen2::roxygenise(pkg$path, roclets, load_code = load_code)
3: force(code)
2: withr::with_envvar(r_env_vars(), roxygen2::roxygenise(pkg$path, 
       roclets, load_code = load_code))
1: devtools::document()
```

Session info:

``` r
sessioninfo::session_info(include_base = TRUE)
#> - Session info ----------------------------------------------------------
#>  setting  value                       
#>  version  R version 3.6.1 (2019-07-05)
#>  os       Windows 10 x64              
#>  system   x86_64, mingw32             
#>  ui       RTerm                       
#>  language (EN)                        
#>  collate  English_United States.1252  
#>  ctype    English_United States.1252  
#>  tz       Europe/Berlin               
#>  date     2019-09-13                  
#> 
#> - Packages --------------------------------------------------------------
#>  ! package     * version date       lib source        
#>    assertthat    0.2.1   2019-03-21 [1] CRAN (R 3.6.0)
#>    base        * 3.6.1   2019-07-05 [?] local         
#>    cli           1.1.0   2019-03-19 [1] CRAN (R 3.6.0)
#>  P compiler      3.6.1   2019-07-05 [2] local         
#>    crayon        1.3.4   2017-09-16 [1] CRAN (R 3.5.1)
#>  P datasets    * 3.6.1   2019-07-05 [2] local         
#>    digest        0.6.20  2019-07-04 [1] CRAN (R 3.6.0)
#>    evaluate      0.14    2019-05-28 [1] CRAN (R 3.6.0)
#>  P graphics    * 3.6.1   2019-07-05 [2] local         
#>  P grDevices   * 3.6.1   2019-07-05 [2] local         
#>    highr         0.8     2019-03-20 [1] CRAN (R 3.6.0)
#>    htmltools     0.3.6   2017-04-28 [1] CRAN (R 3.5.1)
#>    knitr         1.24    2019-08-08 [1] CRAN (R 3.6.1)
#>    magrittr      1.5     2014-11-22 [1] CRAN (R 3.5.1)
#>  P methods     * 3.6.1   2019-07-05 [2] local         
#>    Rcpp          1.0.2   2019-07-25 [1] CRAN (R 3.6.1)
#>    rmarkdown     1.15    2019-08-21 [1] CRAN (R 3.6.1)
#>    sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.6.0)
#>  P stats       * 3.6.1   2019-07-05 [2] local         
#>    stringi       1.4.3   2019-03-12 [1] CRAN (R 3.6.0)
#>    stringr       1.4.0   2019-02-10 [1] CRAN (R 3.6.0)
#>  P tools         3.6.1   2019-07-05 [2] local         
#>  P utils       * 3.6.1   2019-07-05 [2] local         
#>    withr         2.1.2   2018-03-15 [1] CRAN (R 3.5.1)
#>    xfun          0.9     2019-08-21 [1] CRAN (R 3.6.1)
#>    yaml          2.2.0   2018-07-25 [1] CRAN (R 3.5.1)
#> 
#> [1] C:/Users/inp099/Documents/R/win-library/3.6
#> [2] C:/Program Files/R/R-3.6.1/library
#> 
#>  P -- Loaded and on-disk path mismatch.
```

<sup>Created on 2019-09-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by gaborcsardi** [comment]
This is a bug in https://github.com/r-lib/roxygen2/commit/7a8da921c1a4c8030f52771fb37202c4d6467a0d and will be fixed in #908 I believe. If we don't merge that soon, then I can also fix this separately.

**Comment by gaborcsardi** [comment]
@hadley: fyi, after #908, the error is this:
```
Updating ggstatsplot documentation
Error in if (docstring == "") return("") : argument is of length zero
```
coming from here: https://github.com/r-lib/roxygen2/blob/46484f7a55e7d2ba9ace366431090bc9c681c599/R/object-rc.R#L47
